### ISSUE TITLE
Fix backwards compatibility for TL_LANG.LNG and TL_LANG.CNT

### ISSUE THREAD

**Issue by ausi** [contao/contao#4035]
### Affected version(s)

4.12

### Description

Additionally to #3716 we should also fix access to the `$GLOBALS['TL_LANG']['LNG']` and `$GLOBALS['TL_LANG']['CNT']` globals array after calling `System::loadLanguageFile('languages')` and `System::loadLanguageFile('countries')`

**Comment by aschempp** [comment]
Just found this in my code (simplified for readability), which we might need to fix as well:

```
echo $this->translator->trans('CNT.DE', [], 'contao_countries');
```

**Comment by ausi** [comment]
@aschempp you probably meant `CNT.de` (lowercase country code), right?

https://github.com/contao/contao/blob/6af1e0f1a670b84d1c4d1d788695b63d279ed21e/core-bundle/src/Resources/contao/languages/de/countries.xlf#L228-L231

**Comment by ausi** [comment]
Closed in favor of #4043

**Comment by aschempp** [comment]
> @aschempp you probably meant `CNT.de` (lowercase country code), right?

yes 🙈
